src/utils/timeUtil.ts

```diff
@@ -0,0 +1,5 @@
+export const timeUtil = {
+  sleep(ms: number): Promise<void> {
+    return new Promise(resolve => setTimeout(resolve, ms));
+  }
+};
```

Comment:
A tip, instead of doing this, you could use an out of the box async timeout function.

```ts
import { setTimeout } from 'timers/promises';
await setTimeout(500);
```